public/app/features/search/page/components/SearchResultsTable.tsx

```diff
@@ -141,14 +144,42 @@ export const SearchResultsTable = React.memo(
         return (
           <div key={key} {...rowProps} className={className}>
             {row.cells.map((cell: Cell, index: number) => {
+              const href = onClickItem ? url : undefined;
+
+              let userProps: { href?: string; onClick?: (event: React.MouseEvent<HTMLElement>) => void } = {
+                href,
+                onClick: onClickItem,
+              };
```

Comment:
```suggestion
              let userProps = {
                href,
                onClick: onClickItem,
              };
```
No need to declare types here, they'll be inferred form the initial values.